tests/mocks/status.py

```diff
@@ -17,7 +16,6 @@ def mock_status(
     scheduled_status = Status(
         mission_status=mission_status,
         mission_in_progress=mission_in_progress,
-        current_mission_instance_id=current_mission_instance_id,
```

Comment:
We should make sure that the mock contains an ID on the step. Is this done? If not, and the tests are not failing, we probably should extend the test to cover this.

Reply:
There is not an ID field in Status anymore. The id can be found in current_task.id.